rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+comment_width = 100
```

Comment:
I think I prefer 80 for this. My screen is ~170 cols wide, so when I split the window to view two files at once I can see the whole doc comment anyways. Any objections to putting this to 80?

Reply:
No I'm with you on the skinny width! I'll see how it goes. Do you want me to make the max_width 80 as well or just comments?

Reply:
`max_width = 90 and comment_width = 80` looks pretty good.

Reply:
Just comments I would say, most people leave the default for Rust, which is 100?